src/app/first-time-login-dialog/first-time-login-dialog.component.ts

```diff
@@ -0,0 +1,61 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { OAuthService } from 'angular-oauth2-oidc';
+import { Store } from '@ngrx/store';
+import { Router } from '@angular/router';
+import { AppConfigService } from '../app-config.service';
+import { KubernetesClientService } from '../core/kubernetes-client.service';
+import { faAdd, faSitemap } from '@fortawesome/free-solid-svg-icons';
+import { FormControl } from '@angular/forms';
+
+const hideFirstTimeLoginDialogKey = 'hideFirstTimeLoginDialog';
```

Comment:
Could you use this key in the tests as well?

Reply:
> On this screen both modals are shown:
> 
> <img alt="image" width="1392" src="https://user-images.githubusercontent.com/1065341/155109536-466ad633-bfc6-4ff7-a9d4-28e80918e0ff.png">
> 
> Maybe the new modal dialog should only be shown on the "start page"?

I found a solution for that ;).